packages/suite/src/views/wallet/send/components/Options/BitcoinOptions/BitcoinOptions.tsx

```diff
@@ -85,6 +89,13 @@ export const BitcoinOptions = () => {
         setDraftSaveRequest(true);
     };
 
+    const blockchain = useSelector(selectNetworkBlockchainInfo(network.symbol));
+
+    const locktime = watch('bitcoinLockTime');
+
+    const broadcastDisabled =
+        locktime !== undefined && Number(locktime) > blockchain.blockHeight + 1;
```

Comment:
could be simplified as 
```suggestion
      Number(locktime) > blockchain.blockHeight + 1;
```

Reply:
I am not really fun of implicit conversions. `Number(undefined` => `0`. Even it may have sense here, I prefer to be consistent and not doing them at all as they may be source of bugs and unintentional behavior.